azkaban-common/src/main/java/azkaban/executor/ExecutorManager.java

```diff
@@ -79,6 +79,7 @@
       "azkaban.activeexecutor.refresh.milisecinterval";
   private static final String AZKABAN_ACTIVE_EXECUTOR_REFRESH_IN_NUM_FLOW =
       "azkaban.activeexecutor.refresh.flowinterval";
+  private static final int AZKABAN_MAX_NUM_CONCURRENT_FLOW = 30;
```

Comment:
can we make this configurable?

Reply:
is 30 too high?

Reply:
Made it configurable. 30 might hight, but we may set a high number initially, and decrease it gradually, in case break something.